Doc/library/pdb.rst

```diff
@@ -321,9 +321,12 @@ can be overridden by the local file.
    argument must be an identifier, ``help exec`` must be entered to get help on
    the ``!`` command.
 
-.. pdbcommand:: w(here)
+.. pdbcommand:: w(here) [count]
 
-   Print a stack trace, with the most recent frame at the bottom.  An arrow (``>``)
+   Print a stack trace, with the most recent frame at the bottom.  if *count*
+   is 0, print the current frame entry. If *count* is negative, print the least
+   recent - *count* frames. If *count* is positive, print the most recent
+   *count* frames.  An arrow (``>``)
    indicates the current frame, which determines the context of most commands.
```

Comment:
This needs a versionchanged 3.14.

We will backport this, because it's needed for a bug fix, the purpose is not to introduce a new feature in previous versions. The doc changes should not be in the backports, where the feature is not public.